proto/cosmos/bank/v1beta1/bank.proto

```diff
@@ -43,19 +43,6 @@ message Output {
       [(gogoproto.nullable) = false, (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"];
 }
 
-// Supply represents a struct that passively keeps track of the total supply
-// amounts in the network.
-message Supply {
```

Comment:
This is a proto-breaking change. I'm okay to remove it in the v1beta2 package but not here.

I'm adding it back in the migration PR, and adding a deprecated comment to it.

Reply:
We can keep it in the proto files even though it's not used for now. Deprecated is the right step.